internal/cmd/gocdk/init_test.go

```diff
@@ -198,8 +198,9 @@ func TestInferModulePath(t *testing.T) {
 					t.Error(err)
 				}
 			}()
-
-			pctx := newTestProcessContext(dir)
+			srcDir := filepath.Join(dir, "src")
+			os.Mkdir(srcDir, 0777)
+			pctx := newTestProcessContext(srcDir)
 			gopath, cleanup := tc.testGOPATH(dir)
 			defer cleanup()
 			pctx.env = []string{gopath}
```

Comment:
Later down: does this test not assert for the module name? That's the part we care about for this fix, and I'm not sure if we're testing for it.

Reply:
It's not currently asserting for the module name, no. I can add it.